Machine.Specifications.ReSharper.Tests/ReflectionWithSingleProject.cs

```diff
@@ -14,21 +15,26 @@
 using JetBrains.ReSharper.UnitTestFramework;
 using JetBrains.Util;
 using Machine.Specifications.ReSharperProvider;
+using NSubstitute;
 using NuGet.Frameworks;
 using NUnit.Framework;
 
 namespace Machine.Specifications.ReSharper.Tests
 {
     [MspecReferences]
+    [TestNetFramework35]
     public abstract class ReflectionWithSingleProject : BaseTestWithSingleProject
     {
-        private readonly TargetFrameworkId _framework = new TargetFrameworkId(new NuGetFramework(".NETFramework", Version.Parse("4.5")), true);
+        private readonly TargetFrameworkId _framework = new TargetFrameworkId(new NuGetFramework(".NETFramework", new Version(3, 5)), true);
```

Comment:
This is rubbish. The 4.5 and 4.0 versions don't seem to work with the `CompileCs` call below, but 3.5 works? Go figure...